src/components/plots/barPlot.ts

```diff
@@ -179,12 +179,21 @@ export module Plot {
           var secondaryDist = 0;
 
           // if we're inside a bar, distance in both directions should stay 0
-          if (!_Util.Methods.intersectsBBox(queryPoint.x, queryPoint.y, barBBox)) {
+          if (!_Util.Methods.intersectsBBox(queryPoint.x, queryPoint.y, barBBox, 0)) {
```

Comment:
Note the commit comment: "Now test bbox intersection with a tolerance of zero: gives us stable behaviour when moving along the dominant axis at the same secondary axis value." 

It's to make sliding the mouse outside of a bar, but on the same secondary axis value, keep that bar as closest. I.e., when the logic switches from the "within bar" to "within bar's y-range" the resultant bar was different for the same y value when I slid the mouse along x.